README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+I would choose a helper method because it's main function is to change or edit something on the view and the models retrieve and change data in the Database. Also the controller decides which routes to take.
```

Comment:
:+1:

The controller does not decide a route.  It does manage the related models and view.